packages/features/bookings/Booker/components/hooks/useInitialFormValues.ts

```diff
@@ -23,6 +27,107 @@ type UseInitialFormValuesProps = {
   };
 };
 
+//basically it will partially test the input values against the schema. validation are not needed at the initial stage
+export function getFieldSchema(field: Field): ZodSchema<unknown> {
```

Comment:
Why is this `getFieldSchema` needed now?

Reply:
@rajeshj11 Could you please check this. This new change is unnecessarily making things complex.

I don't understand the motivation for this change.